src/services/qix/index.js

```diff
@@ -111,10 +114,24 @@ export default class Qix {
       url += `/identity/${encodeURIComponent(identity)}`;
     }
 
+    let urlParamKeys;
+
     if (reloadURI) {
       url += `?reloadUri=${encodeURIComponent(reloadURI)}`;
```

Comment:
Might be a good idea to use an object with all URL params, and then use https://nodejs.org/api/querystring.html#querystring_querystring_stringify_obj_sep_eq_options to generate the querystring, that way we avoid the `?` and `&` detections, encoding, etc.

Also, I think the `cfg.urlParams.reloadUri` should override the `cfg.reloadUri`.